libs/devices/common/ui/src/lib/views/manage-files/mf-file-list-panel.tsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { FunctionComponent, PropsWithChildren } from "react"
+import styled from "styled-components"
+import { ButtonType, CheckboxSize, IconType } from "app-theme/models"
+import { Button, Checkbox, Typography } from "app-theme/ui"
+import { manageFilesMessages } from "./manage-files.messages"
+
+interface FileListPanelHeaderProps {
+  header: string
+}
+
+export const MfFileListPanelHeader: FunctionComponent<
+  FileListPanelHeaderProps & PropsWithChildren
+> = ({ header, children }) => {
+  return (
+    <FileListPanel>
+      <Typography.H3>{header}</Typography.H3>
+      {children}
+    </FileListPanel>
+  )
+}
+
+export const MfFileListPanelDefaultMode: FunctionComponent<
+  FileListPanelHeaderProps
+> = (props) => {
+  return (
+    <MfFileListPanelHeader {...props}>
+      <AddFileButton message={manageFilesMessages.addFileButtonText.id} />
+    </MfFileListPanelHeader>
+  )
+}
+
+export const MfFileListPanelSelectMode: FunctionComponent<{
+  count: number
+  onAllCheckboxClick: (checked: boolean) => void
+}> = ({ count, onAllCheckboxClick }) => {
+  return (
+    <FileListPanelSelector>
+      <Checkbox
+        size={CheckboxSize.Small}
+        indeterminate
+        onChange={(event) => onAllCheckboxClick(event.target.checked)}
+      />
+      <Typography.P4
+        message={manageFilesMessages.selectionSelectedCount.id}
+        values={{ count }}
+      />
+      <DeleteButton
+        icon={IconType.Trash}
+        message={manageFilesMessages.selectionDelete.id}
+        type={ButtonType.Text}
+      />
+    </FileListPanelSelector>
+  )
+}
+
+const FileListPanel = styled.div`
+  margin: 2.8rem 3.2rem;
+  height: 4rem;
+  display: grid;
+  grid-template-columns: auto auto;
+  justify-content: space-between;
+  align-items: center;
+`
+
+const AddFileButton = styled(Button)`
+  width: 15.6rem;
+`
```

Comment:
This can be achieved with the `ButtonSize.Medium` size prop.

Reply:
308362c2